census/src/main/scala/hmda/census/model/CbsaLookup.scala

```diff
@@ -34,8 +34,10 @@ object CbsaLookup extends CbsaResourceUtils {
     }
   }.toSeq
 
-  val codeMap: Map[String, String] = {
-    values.map(x => (x.cbsa, x.cbsaTitle)).toMap
+  def nameFor(cbsaId: String): String = {
+    values.find(_.cbsa == cbsaId)
```

Comment:
This new approach is interesting, is there a performance improvement gained by switching to `find`?